packages/react-components/src/components/DomainPermissions.tsx

```diff
@@ -58,7 +59,9 @@ const _DomainPermissionsContainer = (props: DomainPermissionsProps): JSX.Element
           <Icon styles={iconPrimaryStyles} iconName={'ControlButtonCameraOn'}></Icon>
           <Icon styles={iconBackDropStyles} iconName={'iconBackdrop'}></Icon>
         </Stack>
-        <Icon styles={iconPrimaryStyles} iconName={'Sparkle'}></Icon>
+        <Stack styles={iconContainerStyles} horizontalAlign={'center'}>
```

Comment:
Do not apply transform here, see comment here: https://teams.microsoft.com/l/message/19:5e150d250f724d1089c949305efdc8cc@thread.skype/1662478489025?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1661978742851&teamName=Azure%20Communication%20Services&channelName=Web%20UI%20-%20Team&createdTime=1662478489025&allowXTenantAccess=false

Reply:
Will be done in follup PR: https://github.com/Azure/communication-ui-library/pull/2307